### PR TITLE
AT__20.01 _001| Table - Icons S,M,L

### PR DESCRIPTION
**Pull request by JMaava**
https://trello.com/c/YMMk3aEC/453-at2001-001-table-icons-sml

### CODE REVIEW

File: cypress/e2e/tableSize.cy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+describe ('<Dashboard>Icons S,M,L', ()=> {
```
Comment (Rom-T) [inline cypress/e2e/tableSize.cy.js:2]: Please remove <> from the describe.

File: cypress/e2e/tableSize.cy.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    it ('Table size S', ()=> {
+        cy.get('a[href="newJob"]').click()
+        cy.get('input#name').type('Project1')
+        cy.get('li[tabindex="0"] span').contains('Freestyle project').click()
+        cy.get('#ok-button').click()
+        cy.get(':nth-child(1) > .model-link').click()
+        
+        cy.get('a[href="/iconSize?16x16"]').contains('S').click()
+        // cy.get('#projectstatus').should('have.class', 'jenkins-table--small')
```
Comment (Rom-T) [inline cypress/e2e/tableSize.cy.js:11]: Please remove comment

**Comment by JMaava** [comment]
Comment and <> removed

**Review by ArinaJur — APPROVED** [review]
ДПЕЬ

**Review by Gepardz — APPROVED** [review]

**Pull request merged**